client/components/commititems.jsx

```diff
@@ -1,13 +1,12 @@
 import React, {Component} from 'react';
-import CommitItem from './commititem';
+import CommitItem from './commitItem';
```

Comment:
actually, the filename is commitItem, with an upper case.
Mine is not the only one with an upper case.
originally, I had it as commit_item, only because this word was difficult to read in all lowercase
- and it seems that someone else refactored my filenames already (from commit_item to commititems).
But they did not refactor other filenames, or if they did, it was not to any consistency.

it is not clear if you want me to refactor this one now in order to use a pull request ?
or if it can be refactored later, along with the rest of the group filename refactoring.